nextflow_schema.json

```
@@ -764,6 +764,17 @@
                     "description": "Maximal number of contigs that are not part of any bin but treated as individual genome.",
                     "help_text": "Contigs that do not fulfill the thresholds of `--min_length_unbinned_contigs` and `--max_unbinned_contigs` are pooled for downstream analysis and reporting, except contigs that also do not fullfill `--min_contig_size` are not considered further."
                 },
+                "bin_min_size": {
+                    "type": "integer",
+                    "default": 0,
```

Catch exceptions in the schema:

```suggestion
                    "default": 0,
                    "minimum": 0,
```

---

I've been writing schemas by hand for the past week so at the moment my brain is full of knowledge about all the options 😅